pkg/registry/core/pod/strategy.go

```diff
@@ -182,7 +178,9 @@ type podEphemeralContainersStrategy struct {
 var EphemeralContainersStrategy = podEphemeralContainersStrategy{Strategy}
 
 func (podEphemeralContainersStrategy) ValidateUpdate(ctx context.Context, obj, old runtime.Object) field.ErrorList {
-	return validation.ValidatePodEphemeralContainersUpdate(obj.(*api.Pod), old.(*api.Pod))
+	// NOTE: ephemeral containers are not able to consume hugepages, so its safe to make this empty.
+	opts := validation.PodValidationOptions{}
```

Comment:
this assumes these validation options will only be hugepages-related in the future... is there a downside to calling the opts helper here?